<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #363636</string>
  </property>
  <widget class="QPushButton" name="addGoalButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeGoalButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>- </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="goalInputBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>660</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="goalList_Widget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>261</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showCompletedGoalButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Show Completed Goal(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Import Goal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Export Goal</string>
   </property>
  </widget>
  <widget class="QLabel" name="Subgoal_LabelTitle">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>160</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28px</string>
   </property>
   <property name="text">
    <string>Subgoal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Goal_LabelTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>160</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28px</string>
   </property>
   <property name="text">
    <string>Goal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addSubGoalButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>560</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeSubGoalButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>560</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="completeGoal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Complete Goal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showSubGoalButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Show Subgoal(s)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="startDateInputBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>580</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="endDateInputBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>580</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>550</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Date</string>
   </property>
  </widget>
  <widget class="QListWidget" name="subgoalList_Widget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>261</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showPointsButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Show Points</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
